<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>337</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>373</width>
    <height>337</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>373</width>
    <height>337</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lil' Piano</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Sounds.qrc">
    <normaloff>:/sounds/yamaha/qt_piano/yamaha/25711.svg</normaloff>:/sounds/yamaha/qt_piano/yamaha/25711.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>371</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="ButtonC">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>150</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>99</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 3px solid black;
background-color:white;
color:black;
font-weight:800;
</string>
         </property>
         <property name="text">
          <string>C</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonD">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>150</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>99</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 3px solid black;
background-color:white;
color:black;
font-weight:800;
</string>
         </property>
         <property name="text">
          <string>D</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonE">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>150</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>99</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 3px solid black;
background-color:white;
color:black;
font-weight:800;
</string>
         </property>
         <property name="text">
          <string>E</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonF">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>150</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>99</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 3px solid black;
background-color:white;
color:black;
font-weight:800;
</string>
         </property>
         <property name="text">
          <string>F</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonG">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>150</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>99</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 3px solid black;
background-color:white;
color:black;
font-weight:800;
</string>
         </property>
         <property name="text">
          <string>G</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonA">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>150</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>99</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 3px solid black;
background-color:white;
color:black;
font-weight:800;
</string>
         </property>
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonH">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>150</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>99</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 3px solid black;
background-color:white;
color:black;
font-weight:800;
</string>
         </property>
         <property name="text">
          <string>H</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonC2">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>150</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>99</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 3px solid black;
background-color:white;
color:black;
font-weight:800;
</string>
         </property>
         <property name="text">
          <string>C</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>391</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
text-shadow:10px 10px black;</string>
    </property>
    <property name="text">
     <string>Lil' Piano</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>373</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QMenu" name="menuPomoc">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <addaction name="actionSterowanie"/>
    <addaction name="actionO_programie"/>
   </widget>
   <addaction name="menuPomoc"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSterowanie">
   <property name="text">
    <string>Sterowanie</string>
   </property>
  </action>
  <action name="actionO_programie">
   <property name="text">
    <string>O programie</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Sounds.qrc"/>
 </resources>
 <connections/>
</ui>
